bqplot/axes.py

```diff
@@ -78,7 +78,7 @@ class Axis(BaseAxis):
     label: string (default: '')
         The axis label
     tick_format: string or None (default: '')
-        The tick format for the axis.
+        The tick format for the axis, for dates use d3 string formatting.
```

Comment:
Actually, we use the same formats as d3. So maybe a more general statement would be better? Or more detailed documentation? Not sure